codesign/lib/src/file_codesign_visitor.dart

```diff
@@ -19,7 +19,7 @@ enum NotaryStatus {
   succeeded,
 }
 
-Logger? logger;
+final Logger log = Logger('codesign');
```

Comment:
This should be moved to its own file so other parts of the code can depend on it without importing all of the codesign logic

Reply:
Good advice, moved